src/lib/token.c

```diff
@@ -263,6 +265,7 @@ CK_RV token_login(token *tok, twist pin, CK_USER_TYPE user) {
     wrappingobject *wobj = &tok->wrappingobject;
     twist wobjauth = tpm_unseal(tpm, sealobjhandle, sealobjauth);
     if (!wobjauth) {
+        rv = CKR_PIN_INCORRECT;
```

Comment:
This looks like a bug and should be split out into a separate patch and can be uploaded independently. I would take that as is.

Reply:
Agreed. Creating another PR for this one.